### DISCUSSION TITLE
MQTT Message Question

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#5258] [General]
If I have a zone with person filter set to 70% threshold. Will Frigate generate mqtt message even if just 1 frame is true when it detects person in that zone?  I am curious if there is an option similar to Motion and Object Filters that affects mqtt message generation.

**Comment by NickM-27** [comment]
mqtt messages will always be sent, it is just a matter of whether or not that object will have the zone attached to it

**Reply by bm55** [reply]
Just trying to find ways to prevent the occitional false alert that I am not able to use the other filters to control. I think suddent change in brightness and movement behind plants or small trees sometimes creates shadows for spit second and triggers alerts - I have mqtt based node red alerts.
